src/model/guild/member.rs

```diff
@@ -352,6 +352,27 @@ impl Member {
     pub fn unban(&self) -> Result<()> {
         http::remove_ban(self.find_guild()?.0, self.user.read().unwrap().id.0)
     }
+
+    /// Returns the permissions for the member.
+    ///
+    /// # Examples
+    /// 
+    /// ```rust,ignore
+    /// // assuming there's a `member` variable gotten from anything.
+    /// println!("The permission bits for the member are: {}", member.permissions().expect("permissions").bits);
+    /// ```
+
+    /// # Errors
+    ///
+    /// Returns a [`ModelError::GuildNotFound`] if the guild the member's in could not be
+    /// found in the cache.
+    ///
+    /// [`ModelError::GuildNotFound`]: enum.ModelError.html#variant.GuildNotFound
+    pub fn permissions(&self) -> Result<Permissions> {
+        let dummy = self.find_guild()?.find().unwrap();
```

Comment:
Needs updating for b4bd771, needs to use the structfield `guild_id` instead of the method `find_guild` now.